homeassistant/components/met_eireann/weather.py

```diff
@@ -54,24 +46,24 @@ async def async_setup_entry(
     coordinator = hass.data[DOMAIN][config_entry.entry_id]
     async_add_entities(
         [
-            MetEireannWeather(
-                coordinator, config_entry.data, hass.config.units.is_metric, False
-            ),
-            MetEireannWeather(
-                coordinator, config_entry.data, hass.config.units.is_metric, True
-            ),
+            MetEireannWeather(coordinator, config_entry.data, False),
+            MetEireannWeather(coordinator, config_entry.data, True),
         ]
     )
 
 
 class MetEireannWeather(CoordinatorEntity, WeatherEntity):
     """Implementation of a Met Éireann weather condition."""
 
-    def __init__(self, coordinator, config, is_metric, hourly):
+    _attr_native_precipitation_unit = LENGTH_MILLIMETERS
+    _attr_native_pressureunit = PRESSURE_HPA
```

Comment:
Does Python have anything to catch this kind of typos?
```suggestion
    _attr_native_pressure_unit = PRESSURE_HPA
```

Reply:
No, as both are technically valid (except not valid in Home Assistant context, but valid in Python).